README.md

```
@@ -8,86 +8,86 @@
-  - Export 3D models to CAD files
-    - [x] `STEP`
-    - [x] `STL`
-    - [x] `3MF`
-    - [x] `ThreeJS`
-    - [x] `OBJ`
+-   Multiple OSes supported
+    -   [x] Windows
+    -   [x] Linux
+    -   [x] macOS
+-   Collaboration on designs
+    -   [x] Versioning of CAD designs using `Git` _(like it's 2024 for real)_
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)